[NUI] remove "_" and refactoring naming to pascal case.

JoogabYun opened the pull request:
### Description of Change ###
<!-- Describe your changes here. -->
 1. Remove class name from api name 
  : View_HasKeyInputFocus() -> HasKeyInputFocus() in Interop.View class
 2. Remove "_" and change to Pascal Case
  : TextEditor_Property_TEXT_COLOR_get();  -> TextColorGet();
 3. Remove "__SWIG_" suffix
  : new_TextEditor__SWIG_0(); -> NewTextEditor();

https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1707

### API Changes ###
<!-- If you have the ACR for changing APIs, put the link of the ACR: -->
 - ACR:

<!--
If you can't make the ACR, List all API changes here (or just put None), example:
Added:
 - SafeBundleHandle Bundle.SafeBundleHandle { get; } // Property
 - void Bundle.AddItem(string key, string value);

Changed:
 - object Bundle.ReceiveItem(string key) => object Bundle.GetItem(string key);
-->

m-macias commented on src/Tizen.NUI/src/internal/Application.cs:
```diff
@@ -1342,70 +1342,70 @@ public static List<Window> GetWindowList()
         internal ApplicationSignal InitSignal()
         {
-            ApplicationSignal ret = new ApplicationSignal(NDalicPINVOKE.Application_InitSignal(swigCPtr), false);
+            ApplicationSignal ret = new ApplicationSignal(NDalicPINVOKE.ApplicationInitsignal(swigCPtr), false);
```
```suggestion
            ApplicationSignal ret = new ApplicationSignal(NDalicPINVOKE.ApplicationInitSignal(swigCPtr), false);
```

m-macias commented on src/Tizen.NUI/src/internal/Application.cs:
```diff
@@ -1342,70 +1342,70 @@ public static List<Window> GetWindowList()
         internal ApplicationSignal TerminateSignal()
         {
-            ApplicationSignal ret = new ApplicationSignal(NDalicPINVOKE.Application_TerminateSignal(swigCPtr), false);
+            ApplicationSignal ret = new ApplicationSignal(NDalicPINVOKE.ApplicationTerminatesignal(swigCPtr), false);
```
```suggestion
            ApplicationSignal ret = new ApplicationSignal(NDalicPINVOKE.ApplicationTerminateSignal(swigCPtr), false);
```

m-macias commented on src/Tizen.NUI/src/internal/Application.cs:
```diff
@@ -1342,70 +1342,70 @@ public static List<Window> GetWindowList()
         internal LowBatterySignalType BatteryLowSignal()
         {
-            LowBatterySignalType ret = new LowBatterySignalType(NDalicPINVOKE.Application_LowBatterySignal(swigCPtr), false);
+            LowBatterySignalType ret = new LowBatterySignalType(NDalicPINVOKE.ApplicationLowbatterysignal(swigCPtr), false);
```
```suggestion
            LowBatterySignalType ret = new LowBatterySignalType(NDalicPINVOKE.ApplicationLowBatterySignal(swigCPtr), false);
```

m-macias commented on src/Tizen.NUI/src/internal/FixedRuler.cs:
```diff
@@ -72,7 +72,7 @@ public override float GetPositionFromPage(uint page, out uint volume, bool wrap)
         public override uint GetPageFromPosition(float position, bool wrap)
         {
-            uint ret = Interop.Ruler.FixedRuler_GetPageFromPosition(swigCPtr, position, wrap);
+            uint ret = Interop.Ruler.FixedrulerGetpagefromposition(swigCPtr, position, wrap);
```
```suggestion
            uint ret = Interop.Ruler.FixedRulerGetPageFromPosition(swigCPtr, position, wrap);
```

m-macias reviewed (commented):
Maybe there is a name collision and it is a reason why it was named that way, but there are multiple instances that use PascalCase incorrectly. I marked only some of them.

dongsug-song commented:
@JoogabYun 
Hello, 
Thank you for your huge job done. 
but as you know, it need get success from checker
please check errors.
thank you.

JoogabYun commented:
@m-macias 
Thank you for your review.
I will check again for the pascal case transformation.

dongsug-song commented:
@JoogabYun 
Hello, 
thank you for your big work.
but could you resolve conflicts?
and also could you get @m-macias 's review again?
thank you.

JoogabYun commented:
@dongsug-song 
I updated patch. and I took care of m-macias's review.
Fixed incorrect Pascal case
ex) ApplicationLowBatterysignal -> ApplicationLowBatterySignal

dongsug-song reviewed (approved):
I think there would be build errors in GBM's image release. 
for that case, I will request for you to fix those errors by adding same APIs which just have different names.
in parrallel we will make some list of these APIs for GBM to change.
thank you.
